src/addon/addon_manager.cpp

```diff
@@ -538,10 +571,11 @@ AddonManager::unmount_old_addons()
 {
   for (auto& addon : m_installed_addons) {
     if (is_old_enabled_addon(addon)) {
-      if (PHYSFS_unmount(addon->get_install_filename().c_str()) == 0)
-      {
-        log_warning << "Could not remove " << addon->get_install_filename() << " from search path: "
-                    << PHYSFS_getLastErrorCode() << std::endl;
+      if (g_config->use_local_path)
+        if (PHYSFS_unmount(addon->get_install_filename().c_str()) == 0)
+        {
+          log_warning << "Could not remove " << addon->get_install_filename() << " from search path: "
+                     << PHYSFS_getLastErrorCode() << std::endl;
       }
     }
```

Comment:
Leave the wrong bracket lining on the lines you didn't write, but add brackets to `g_config->use_local_path` to make the code cleaner

Reply:
I've cleaned this up the best I can. I added brackets.

I think it's ready.